.github/workflows/maven.yml

```diff
@@ -5,9 +5,13 @@ name: Java CI with Maven
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
+      - 'release/**'
```

Comment:
I don't recall ever using release branches in client libraries, why is this change necessary?

Reply:
It's just to avoid changing this file again as done [here](https://github.com/onfido/onfido-java/pull/71/files) to retrofit a change in an older release. Naming of release branch can be changed, anyway it's just to have the possibility to test and deliver changes when they're not on the master branch (i.e. deliver a patch for an older release): we can still use master branch for the _normal_ changes on last release.

Reply:
Ah ok, makes sense 👍